docs/01-Using-Fleet/04-Adding-hosts.md

```diff
@@ -25,9 +26,22 @@ fleetctl package --type pkg --fleet-url=[YOUR FLEET URL] --enroll-secret=[YOUR E
 
 When you install the generated osquery installer on a host, this host will be automatically enrolled in the specified Fleet instance.
 
+### Signing installers
+
+  >**Note:** Windows signing is not yet supported within `fleetctl` (working on that this week), but the resulting MSI could be signed after building.
+
+The `fleetctl package` command provides suppport for signing and notarizing macOS osquery packages via the
```

Comment:
```suggestion
The `fleetctl package` command provides suppport for signing and notarizing macOS osquery installers via the
```

I propose updating "packages" to "installers" here. In the documentation, I'd like to consistently refer to the packages as "Fleet's osquery installers," "macOS installers," "macOS osquery installers," etc.